hardware_interface/test/test_handle.cpp

```diff
@@ -38,9 +36,7 @@ TEST(TestHandle, command_interface)
 TEST(TestHandle, state_interface)
 {
   double value = 1.337;
-  StateInterface interface {
-    JOINT_NAME, FOO_INTERFACE, &value
-  };
+  StateInterface interface{JOINT_NAME, FOO_INTERFACE, &value};
```

Comment:
```suggestion
  StateInterface interface {
    JOINT_NAME, FOO_INTERFACE, &value
  };
```
formatting suggestion